src/redux/reducers/index.js

```diff
@@ -146,9 +146,11 @@ const createNote = (text) => {
 export const collectionsReducer = (state = preloadCollections, action) => {
   switch (action.type) {
     case actionTypes.CLEAR_PATIENT_DATA: {
-      const newDefaultCollection = createCollection();
+      const { id } = defaultCollection;
+      defaultCollection = createCollection();
+      defaultCollection.id = id;
       return {
-        [newDefaultCollection.id]: newDefaultCollection,
+        [defaultCollection.id]: defaultCollection,
```

Comment:
These parts of redux probably get refactored once we pause and purge the store on logout. There shouldn't be any collections data after logging out.

Reply:
TODO for the next PR relating to pause and purge